packages/playground/src/components/execution/mocks/mockInput.ts

```diff
@@ -0,0 +1,87 @@
+//simple data for rendering
+export const inputData = [
```

Comment:
mock data should be in `playground/src/pages`, not in `components` folder

Reply:
move to  mocks directory in pages folder

Reply:
change mockdata into pages